cumulus/client/cirrus-executor/src/processor.rs

```diff
@@ -206,17 +210,43 @@ where
 
 		// The applied txs can be fully removed from the transaction pool
```

Comment:
BTW, is this comment a TODO? It doesn't seem to point to something specific.

Reply:
Yes, a TODO, will update it!